internal/controllers/s3userclaim/handler.go

```diff
@@ -39,9 +39,10 @@ import (
 
 type Reconciler struct {
 	client.Client
-	scheme    *runtime.Scheme
-	logger    logr.Logger
-	rgwClient *admin.API
+	uncachedReader client.Reader
```

Comment:
@therealak12 
I've used uncached reader for quota calculations. Is it right?

Reply:
The reason I've used an uncached reader in the webhook is to ensure no resource is missed when checking the quota excesses.

We can use the default cached client as updating the status is not as critical as validation.

Reply:
Right. I suggest keeping it for now since the user could be sure of the shown quota.